Dev-Corner/images/toolshed_instance.graphml

```diff
@@ -0,0 +1,982 @@
+	c0.739-0.066,1.479-0.145,2.217-0.229c0,0.942,0,1.887,0,2.83c-0.736,0.092-1.478,0.177-2.217,0.252
+	C26.184,49.009,26.184,48.06,26.184,47.109z M26.184,46.051c-0.002-0.944-0.002-1.888-0.002-2.832
+	c0.739-0.06,1.48-0.127,2.219-0.202c0,0.938,0,1.873,0,2.811C27.662,45.912,26.923,45.986,26.184,46.051z M26.182,42.162
+	c0-0.95-0.002-1.9-0.002-2.85c0.74-0.052,1.48-0.109,2.219-0.176c0.002,0.943,0.002,1.887,0.002,2.83
+	C27.662,42.039,26.921,42.105,26.182,42.162z M26.18,38.253c0-0.95,0-1.9-0.002-2.852c0.742-0.041,1.482-0.093,2.221-0.146
+	c0,0.942,0,1.887,0,2.829C27.66,38.15,26.92,38.203,26.18,38.253z M26.178,34.345c0-0.949,0-1.898,0-2.852
+	c0.74-0.034,1.481-0.073,2.221-0.117c0,0.943,0,1.887,0,2.83C27.659,34.258,26.918,34.305,26.178,34.345z M26.177,30.437
+	c0-0.949,0-1.9-0.001-2.85c0.741-0.027,1.481-0.059,2.221-0.092c0,0.943,0.002,1.888,0.002,2.83
+	C27.659,30.367,26.918,30.404,26.177,30.437z M26.176,26.529c-0.001-0.942-0.001-1.888-0.001-2.832
+	c0.742-0.018,1.482-0.039,2.222-0.063c0,0.938,0,1.873,0,2.811C27.657,26.476,26.917,26.503,26.176,26.529z M26.174,22.64
+	c0-0.951-0.001-1.901-0.001-2.851c0.741-0.01,1.483-0.022,2.224-0.035c0,0.943,0,1.886,0,2.831
+	C27.657,22.605,26.915,22.623,26.174,22.64z M8.769,15.881c0,0.95,0,1.9-0.001,2.85c-0.741-0.008-1.482-0.018-2.223-0.028
+	c0-0.943,0-1.887,0-2.83C7.286,15.876,8.028,15.878,8.769,15.881z M6.54,50.758c0.738,0.097,1.478,0.183,2.218,0.258
+	c0,0.95,0,1.901,0,2.853c-0.741-0.084-1.48-0.178-2.218-0.28C6.54,52.646,6.54,51.701,6.54,50.758z M26.184,53.869
+	c0-0.95,0-1.899,0-2.853c0.739-0.075,1.479-0.163,2.217-0.259c0.002,0.941,0.002,1.889,0.002,2.83
+	C27.663,53.693,26.925,53.785,26.184,53.869z"/&gt;
+&lt;path id="highlight_2_" opacity="0.17" fill="#FFFFFF" enable-background="new    " d="M0,0.943h5.497c0,0,6.847-0.943,11.974-0.943
+	C22.6,0,29.48,0.943,29.48,0.943h5.496v41.951c0,0-12.076-0.521-18.623-2.548C9.807,38.32,0,30.557,0,30.557V0.943z"/&gt;
+&lt;/svg&gt;
+</y:Resource>
+      <y:Resource id="3" type="java.awt.image.BufferedImage">iVBORw0KGgoAAAANSUhEUgAAASQAAABLCAYAAADZLPrXAAAat0lEQVR42u1dCXQcxZluyQaMwRfY&#13;
+4AOfCNuyZUkzkuY+uqe7p3suyTaYwyHmGYJ57OKQR8LxHllwAuEwhBcCYUmAZUMeJBhIlsO2ZNjY&#13;
++ICQddhlwWAwd4AFDNhcwbKu/f9W1bgtZqar59LIru+9ehrN9HR39VR99d8lCBwcHBwcHBwcHBwc&#13;
+HBwcHBwcHBwcHBwcHBxZUcXQODg4ODg4OA5VVIuiOHzNmjXDWL+AxzY1NR2xZMmSYYeipDC3psa7&#13;
+MJXqs2ptyWSfHo1ejd8hz4KDg8MukIBM/45vbm4+La7rt8jh8B9TsdjOqCz/nyJJe6F9YbRI5DNd&#13;
+Vd9LxmIvyaL4R1WWb/a53Wd7nM5aM0khuR0Kz2fOnDnNhHD2I+lka63J5DdRRfkRJyQODvtISzL1&#13;
+8+dHNEV5OKaq3YtaW/uwwQTrhNYFrRtaT5bWTSbpfvwOTlr8C2S1KxQIrKqZOvVkPP+qVauGD2XJ&#13;
+iRMSB0c5JCMgjdZEghIQTrbeXBPORuuGCdyDkzimabs9LtdKuNywoTpROSFxcJQQIL38cyoeR4mm&#13;
+p4gklK0ZxATqXR9c84Ha6dMn9fX1DSmvFCckDo4SwOVyHZ/QtJfJ5OopMRENbL0ojSnh8BsZ7Fac&#13;
+kDgOYVUEBjt6fSqtDdYkRIlEi0RcsWi002pSlbIhIcnh8AuckDgOK3iam1sXzJu3tNKas65uEbnF&#13;
+cqorVWGXK4hkQIzUfZyQOCFxlAkKjHQ00i5ua6u4hvflc7v
```

Comment:
ToolShed - the Galaxy "Appstore" to make people understand what this is and that everyone can get tools?